fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -495,6 +506,7 @@ def oot_dl_writer_register():
 	bpy.types.Scene.ootDLImportUseCustomPath = bpy.props.BoolProperty(
 		name = "Use Custom Path")
 
+	bpy.types.Scene.ootDLImportAll = bpy.props.BoolProperty(name = "Import All DLs", default = False)
```

Comment:
add this to `OOT_Properties` instead, and access it like `scene.fast64.oot.importAllDL` (or whatever name you like for `importAllDL`)

Reply:
Why? This is set up exactly the same way as all the other controls for importing and exporting DLs (and the other import/export tools too). Why should this one be handled differently?

Reply:
We only introduced the .fast64 property groups recently so old ones haven't been migrated (yet), but new ones should go there

Why did we introduce these property groups, because it's better practice to group custom properties, less chance for name clashes and more organized


Not that I like having what's essentially an operator property as a scene property at all anyway but that's probably too late to change

Reply:
I mean sure, that's a better place to put them, but I think it's crazy to put one of the controls for an operator in one place and leave the rest of them in a different place. This isn't a new category of control, this is exactly like the other DL import / export controls. When the other ones get "migrated", whoever does that can migrate this one too.